portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -183,13 +183,52 @@ definition {
 	}
 
 	@description = "Verify that options can be rearranged"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test OptionsCanBeRearranged {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132207 OptionsCanBeRearranged pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Single Selection");
+
+		Type(
+			key_rowNumber = "1",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 1");
+		
+		Type(
+			key_rowNumber = "2",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 2");
```

Comment:
Hi @PauloVAS,

Instead of typing directly, you can use the `FormFields.editOptionsRow` macro.

For this test scenario, it would be something like this:

```
FormFields.editOptionsRow(
	optionName = "Option1",
	optionValue = "Option 1",
	rowNumber = "1");

FormFields.editOptionsRow(
	optionName = "Option2",
	optionValue = "Option 2",
	rowNumber = "2");
```
You can find an example [here](https://github.com/liferay/liferay-portal/blob/cc5ccd2a3e82780cc87de95c93973ee585d632ab/portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase#L1800-L1807).